packages/ui/src/text/stories/index.story.tsx

```diff
@@ -0,0 +1,52 @@
+import { type Meta, type StoryObj } from '@storybook/react';
+import '@wordpress/theme/design-tokens.css';
+import { Text } from '../text';
+
+const meta: Meta< typeof Text > = {
+	title: 'Design System/Components/Text',
+	component: Text,
+	tags: [ 'status-experimental' ],
+};
+export default meta;
+
+type Story = StoryObj< typeof Text >;
+
+export const Default: Story = {
+	args: {
+		children: 'The quick brown fox jumps over the lazy dog',
+		fontFamily: 'body',
+		fontSize: 'md',
+		fontWeight: 'regular',
+		lineHeight: 'md',
+	},
+};
+
+export const Heading: Story = {
+	args: {
+		children: 'Heading Text',
+		fontFamily: 'heading',
+		fontSize: 'xl',
+		fontWeight: 'medium',
+		lineHeight: 'lg',
+	},
+};
+
+export const Body: Story = {
+	args: {
+		children: 'Body text with regular weight and medium size.',
+		fontFamily: 'body',
+		fontSize: 'md',
+		fontWeight: 'regular',
+		lineHeight: 'md',
+	},
+};
```

Comment:
These combinations did give me pause to consider if this is evidence of the tokens being "primitive", with "semantic" tokens referencing the combinations of those aspects toward application of higher-level concept of "heading" and "body" content. 

There's some suggestion to this effect in examples in the DTCG specification: https://www.designtokens.org/tr/2025.10/format/#typography-component-references

My hunch is that the way we're using these values is still semantic, and something we'll need to support. And maybe, on top of that, we could explore pre-composed combinations as a separate set of tokens referencing these.

Reply:
We should definitely support variants as mentioned, and we should probably assign those styles with a low-enough specificity that will allow consumers to override indivual aspects without struggling (although we should encourage sticking to variables as much as possible)